### PR TITLE
Add success stories

### PR DESCRIPTION
**Pull request by paragsahu**

### CODE REVIEW

**Comment by paragsahu** [comment]
Please review this, @casey-collab

File: hindi/bollywood.md [hunk 1]
```diff
@@ -29,4 +29,6 @@ Art of Living [Know More](art-of-living/art-of-living.md)
 
 Click [Here](king/king.md) to read the story of a great King.
 
-To know about real india hero click [Bhagat Singh](Bhagat_Singh/bhagat_singh.md)
\ No newline at end of file
+To know about real india hero click [Bhagat Singh](Bhagat_Singh/bhagat_singh.md)
+
+[Read success stories](http://www.thesecret.tv/all-stories/)
```
Comment (casey-collab) [inline hindi/bollywood.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: hindi/bollywood.md (hunk 1 above)
Comment (paragsahu) [inline hindi/bollywood.md]: Its working

**Pull request closed without merging**